File: isPalindrome.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if (x<0): return False
        y = [int(i) for i in str(x)]
        for j in range(len(y)):
            print(j,y[j] , y[len(y)-1-j])
            if (y[j] !=y[len(y)-1-j]):
                return False
        return True
print (Solution().isPalindrome(121))

File: isPalindrome.js
var isPalindrome = function(x) {
    let y = [];   
    while(x>0){
        y.unshift(x%10);  
        x=x/10|0; 
    }
    // const z = String(x).split("").map(ele=>parseInt(ele));
    // console.log(11111111, z)

    console.log(y)
    let ans = true;
    if (Math.sign(x) == -1) return false

      for(let i=0; i < y.length; i++)
    {
        if (y[y.length-1-i] !== y[i]) { ans = false; break}
    }
    return ans;
};

console.log(isPalindrome(223));

